domains/authorization/Authorization.API/API/Authorization/Controllers/ConsentController.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using API.Authorization._Features_;
+using API.ValueObjects;
+using Asp.Versioning;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace API.Authorization.Controllers;
+
+[ApiController]
+[ApiVersion(ApiVersions.Version20230101)]
+public class ConsentController : ControllerBase
+{
+    private readonly IMediator _mediator;
+    private readonly EntityDescriptor _entityDescriptor;
+
+    public ConsentController(IMediator mediator, EntityDescriptor entityDescriptor)
+    {
+        _mediator = mediator;
+        _entityDescriptor = entityDescriptor;
+    }
+
+    /// <summary>
+    /// Grants consent.
+    /// </summary>
+    [HttpPost]
+    [Authorize(Policy = "SubTypeUser")]
```

Comment:
This policy isn't defined anywhere as far as i know. Maybe delete this and add a subtask to the story?

Reply:
The policy is defined in program.cs
```
builder.Services.AddAuthorizationBuilder()
    .AddPolicy("SubTypeUser", policy => policy.RequireClaim("sub_type", "user")
);
```

Reply:
Ok, jeg tænker at den forsvinder når vi opgraderer til nyeste version af TokenValidation, så må vi lige have den tilføjet der i stedet.